template/config/variables/development.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+/**
+ * This file should define variables that webpack should make available to 
+ * your application when running in development mode (NODE_ENV === 'develop')
+ * 
+ * It should not contain any sensitive information. See README.
+ */
+module.exports = {
+  NODE_ENV: 'development'
```

Comment:
Since we now aggregate variables and stingify their values in `/build/utils`, we don't have to stringify them here anymore - so no more `'" "'`double quotes!